sdk/python/kfp/cli/output.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from enum import Enum, unique
+
+from tabulate import tabulate
+
+
+@unique
+class OutputFormat(Enum):
+    table = "table"
+    json = "json"
+
+
+def print_output(data, headers, output_format, table_format='simple'):
```

Comment:
nit: it'll be great if a docstring is available here :) so that we can easily enhance/maintain this piece of code in the future.

Also good to have type hints.

Reply:
Added type hints and docstring.